src/patchers/@nextcloud/axios.js

```diff
@@ -28,10 +28,18 @@ axios.interceptors.request.use((config) => {
 	return config
 }, (error) => Promise.reject(error))
 
+let upgradeInterceptorHasBeenTriggeredOnce = false
+
 axios.interceptors.response.use((response) => response, (error) => {
 	if (error?.response?.status === 401) {
 		window.TALK_DESKTOP.logout()
 	}
+	if (error?.response?.status === 426) {
+		if (!upgradeInterceptorHasBeenTriggeredOnce) {
+			upgradeInterceptorHasBeenTriggeredOnce = true
+			window.TALK_DESKTOP.showUpgrade()
+		}
+	}
```

Comment:
**praise:** HTTP interceptor

Thank you for implementing this approach for handling 426!